mithril-aggregator/Cargo.toml

```diff
@@ -17,7 +17,7 @@ cloud-storage = "0.11.0"
 config = "0.13.1"
 flate2 = "1.0.23"
 hex = "0.4.3"
-mithril-common = { path = "../mithril-common" }
+mithril-common = { path = "../mithril-common" , default-features = false}
```

Comment:
Why do we need to deactivate the default features? it should work without deactivating them. Now it doesn't really make a different, but I don't see the need either.

Reply:
You're right, this is something that should not have been committed :heavy_check_mark: